go/hyperkit.go

```diff
@@ -355,6 +358,9 @@ func (h *HyperKit) buildArgs(cmdline string) {
 	if h.DiskImage != "" {
 		a = append(a, "-s", fmt.Sprintf("2:0,virtio-blk,%s", h.DiskImage))
 	}
+	if h.VSock {
+		a = append(a, "-s", fmt.Sprintf("3,virtio-sock,guest_cid=3,path=%s", h.StateDir))
```

Comment:
On Docker for Mac we mostly keep them in the same directory. I leave them as is for now and re-arrange if it becomes an issue